### PR TITLE
Fix php warnings

### PR DESCRIPTION
**Pull request by jfuad**
#2274 I looked a few pages and fixed some notices.

below added by bradymiller:
Up For Grabs demo for this PR is here: https://www.open-emr.org/wiki/index.php/Development_Demo#Beta_-_Up_For_Grabs_Demo

### CODE REVIEW

File: library/menu/src/PatientMenuRole.php [hunk 1]
```diff
@@ -55,9 +55,14 @@ public function getMenu()
...
-            $menu_obj -> url = $this->getAbsoluteWebRoot($menu_obj -> url);
+            if (property_exists($menu_obj, 'url')) {
+                $menu_obj -> url = $this->getAbsoluteWebRoot($menu_obj -> url);
+            }
             if (!empty($menu_obj->children)) {
                 foreach ($menu_obj->children as $menu_obj) {
+                    if (property_exists($menu_obj, 'url')) {
+                        $menu_obj -> url = $this->getAbsoluteWebRoot($menu_obj -> url);
+                    }
                     $menu_obj -> url = $this->getAbsoluteWebRoot($menu_obj -> url);
```
Comment (bradymiller) [inline library/menu/src/PatientMenuRole.php]: Is the above line supposed to be removed?

**Comment by bradymiller** [comment]
Up For Grabs demo for this PR is here:
https://www.open-emr.org/wiki/index.php/Development_Demo#Beta_-_Up_For_Grabs_Demo
(so folks can test this PR easily)

File: library/menu/src/PatientMenuRole.php (hunk 1 above)
Comment (jfuad) [reply library/menu/src/PatientMenuRole.php]: Yes, I forgot the line 66. The entry "Modules" in \interface\main\tabs\menu\menus\patient_menus\standard.json hasn't the 'url' property.

**Comment by bradymiller** [comment]
hi @jfuad ,
These code changes look good and straightforward, so bringing into codebase. Thanks for the fixes! -brady

**Pull request merged**